development/java/Rubberduck.Parsing/Grammar/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        mavenCentral();
+    }
+
+    dependencies {
+        classpath group: 'org.antlr', name: 'antlr4', version: '4.7.2'
+    }
+}
+
+plugins {
+    id 'java'
+}
+
+group 'com.rubberduckvba.rubberduck.parsing'
+version '1'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'org.antlr', name: 'antlr4-runtime', version: '4.7.2'
+}
+
+def grammerCodeGenDest = "${projectDir}/src/main/gen"
+def grammarSource = "${projectDir}/../../../../Rubberduck.Parsing/Grammar/"
```

Comment:
this path traversal is making me somewhat queasy...

Reply:
@Vogel612 
Indeed. However, I am also unable unsure of a better way.